lib/elixir/lib/kernel.ex

```diff
@@ -1206,13 +1228,22 @@ defmodule Kernel do
   @doc """
   Gets the element at the zero-based `index` in `tuple`.
 
+  It raises `ArgumentError` when `tuple` is empty,
+  or when `index` is a negative number or it is out of range of the tuple elements.
```

Comment:
If the tuple is empty or the index is negative it is automatically out of range, I think it's enough to say that.